src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -419,6 +419,14 @@ protected function getEnv($name)
         if (false !== $env = getenv($name)) {
             return $this->envCache[$name] = $env;
         }
+        if (false !== $i = strpos($name, '@')) {
```

Comment:
Yep, that's what I thought, and that would require a class, which means a service, thus a service definition. But the container comes without any service preconfigured...